### ISSUE TITLE
Compare also against swiper-isearch in README

### ISSUE THREAD

**Issue by a13** [raxod502/ctrlf#39]
here https://github.com/raxod502/ctrlf#why-not-swiper you didn't mention `swiper-isearch`, which was introduced a year ago

**Comment by raxod502** [comment]
Reasonable.

**Comment by aspiers** [comment]
Also, having used swiper for quite a while, and having only just switched to `ctrlf` today (BTW I love it already!), I find the argument in that section unnecessarily weak.  It says:

> The selling point of Swiper is that it shows you an overview of the matches. Ask yourself: when was the last time you actually got anything useful out of that overview?

Actually, I find that overview incredibly useful, multiple times pretty much every day.  However `consult-line` is a perfect substitute (and also proof that at least one other person finds that functionality useful).  So I suggest rephrasing that argument.

**Comment by aspiers** [comment]
Maybe also worth noting that the weakness of that argument delayed my switch to `ctrlf` by several months.  Now I'm feeling regret for the delay ;-)

**Comment by a13** [comment]
> Also, having used swiper for quite a while

The question here is not about `swiper` but about `swiper-isearch`, which does everything (I suppose) `ctrlf` can do and even more.

> Now I'm feeling regret for the delay ;-)

what for?

**Comment by a13** [comment]
> Maybe also worth noting that the weakness of that argument delayed my switch to `ctrlf` by several months. Now I'm feeling regret for the delay ;-)

taking into account that there is (and existed before `ctrlf` was "invented") `swiper-isearch`, that section should be empty since

> Furthermore, Swiper constrains itself almost exclusively to line-based search by design, which makes it unsuitable to the task of quick movement within a line or movement to a commonly occurring search string.

argument is not valid either

**Comment by raxod502** [comment]
> I find the argument in that section unnecessarily weak

Fair enough. I will note that other people have said they agree with it, so I think whether or not the overview is helpful depends on the individual person. It makes sense to make this subjectivity explicit in the README.

> Now I'm feeling regret for the delay ;-)

What information could have been available that would have communicated the changes of CTRLF that matter to you personally? Or was it just that the note about the overview not being helpful seemed very wrong to you, and hence you didn't look further?

**Comment by aspiers** [comment]
[**@raxod502**](https://github.com/raxod502) commented on [December 21, 2020 11:01 PM](https://github.com/raxod502/ctrlf/issues/39#issuecomment-749244385 "2020-12-21T23:01:18Z - Replied by Github Reply Comments"):
> > I find the argument in that section unnecessarily weak
> 
> Fair enough. I will note that other people have said they agree with it, so I think whether or not the overview is helpful depends on the individual person. It makes sense to make this subjectivity explicit in the README.

Agreed :clap: 

> > Now I'm feeling regret for the delay ;-)
> 
> What information could have been available that would have communicated the changes of CTRLF that matter to you personally?

Well the first thing I really liked was the ability to navigate multiple matches within a single line, but that was before I realised that `swiper-isearch` existed.  The other thing I like is seeing the count of matches right next to the match.  Other than that and the overview, I'm not sure there are huge differences with `swiper-isearch`.

> Or was it just that the note about the overview not being helpful seemed very wrong to you, and hence you didn't look further?

Yeah pretty much, but that was also before I learned that I could get an overview from `occur` (which is kind of crazy and embarrassing given that I've used emacs since 1994).  If #69 was addressed then I would not miss the absence of an overview being displayed in the minibuffer by default.

**Comment by a13** [comment]
> I like is seeing the count of matches right next to the match

like this?

![screenshot](https://user-images.githubusercontent.com/251267/102978051-9106d400-450c-11eb-95ee-35c84dbef75c.png)

**Comment by a13** [comment]
> It makes sense to make this subjectivity explicit in the README.

In such case the phrasing isn't perfect. Something like "I don't use it" would sound more stainless (to me)

**Comment by aspiers** [comment]
[**@a13**](https://github.com/a13) commented on [December 23, 2020 8:50 AM](https://github.com/raxod502/ctrlf/issues/39#issuecomment-750016214 "2020-12-23T08:50:21Z - Replied by Github Reply Comments"):
> > I like is seeing the count of matches right next to the match
> 
> like this?

No, that's not right next to the match.

**Comment by raxod502** [comment]
> right next to the match

I'm assuming you mean this functionality of CTRLF, which displays the match count and index within the buffer, rather than only the minibuffer:

![image](https://user-images.githubusercontent.com/6559064/103177898-2c60b780-4833-11eb-9846-fde35aef537a.png)

**Comment by aspiers** [comment]
Exactly.
